worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala

```diff
@@ -58,8 +58,10 @@ class PushDataHandler extends BaseMessageHandler with Logging {
   var storageManager: StorageManager = _
   var conf: CelebornConf = _
   @volatile var pushDataTimeoutTested = false
+  var _worker: Worker = _
```

Comment:
We should not add Worker as PushHandler's member

Reply:
it's removed